src/main/java/org/candlepin/insights/ApplicationProperties.java

```diff
@@ -48,4 +49,11 @@ public void setPrettyPrintJson(boolean prettyPrintJson) {
         this.prettyPrintJson = prettyPrintJson;
     }
 
+    public boolean isDevMode() {
+        return devMode;
+    }
+
+    public void setDevMode(boolean devMode) {
+        this.devMode = devMode;
+    }
```

Comment:
Can you please set in `rhsm-conduit.properties` to the `DEV_MODE` environment variable (defaulting to false), and document in `README.md`?